app/controllers/wikis_controller.rb

```diff
@@ -9,18 +13,31 @@ def show
 
   def new
     @wiki = Wiki.new
+    begin
```

Comment:
The issue is that I wanted different "flash" messages depending on exactly why `authorize` failed.